doc/manipulating_images/manipulating_images.rst

```diff
@@ -132,6 +132,14 @@ Accessing individual volumes in 4D images
      creates a new image using an existing reference image for the
      metadata.
 
+|
+
+.. topic:: **Examples**
```

Comment:
> For the record, there is a way to include all the examples using a particular function

Indeed, but I wanted to be specific, rather than exhaustive.

The reference documentation of a function already has an exhaustive list.